tests/e2e/kms.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: azure-kms-provider
+  namespace: kube-system
+  labels:
+    tier: control-plane
+    component: azure-kms-provider
+spec:
+  priorityClassName: system-node-critical
+  hostNetwork: true
+  containers:
+    - name: azure-kms-provider
+      image: ${REGISTRY}/${IMAGE_NAME}:${IMAGE_VERSION}
+      imagePullPolicy: IfNotPresent
+      args:
+        - --keyvault-name=${KEYVAULT_NAME}
+        - --key-name=${KEY_NAME}
+        - --key-version=${KEY_VERSION}
+        - -v=2
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 4
+          memory: 2Gi
+      volumeMounts:
+        - name: etc-kubernetes
+          mountPath: /etc/kubernetes
+        - name: etc-ssl
+          mountPath: /etc/ssl
```

Comment:
where do we use this?

Reply:
This is required to access root CA certs. Without this volume kv calls will fail with:
```bash
2021-02-09T01:52:20.524238Z stderr F E0209 01:52:20.523812       1 server.go:50] "failed to encrypt" err="failed to encrypt, error: keyvault.BaseClient#Encrypt: Failure sending request: StatusCode=0 -- Original Error: context deadline exceeded"
```